WcaOnRails/spec/models/merge_people_spec.rb

```diff
@@ -25,14 +25,14 @@
     person = FactoryGirl.create(:person_with_multiple_sub_ids)
     merge_people.person1_wca_id = person.wca_id
     expect(merge_people).to be_invalid
-    expect(merge_people.errors.messages[:person1_wca_id]).to eq ["This person has multiple subIds"]
+    expect(merge_people.errors.messages[:person1_wca_id]).to include "This person has multiple subIds"
   end
 
   it "requires person2 not have multiple subIds" do
     person = FactoryGirl.create(:person_with_multiple_sub_ids)
     merge_people.person2_wca_id = person.wca_id
     expect(merge_people).to be_invalid
-    expect(merge_people.errors.messages[:person2_wca_id]).to eq ["This person has multiple subIds", "Names don't match"]
+    expect(merge_people.errors.messages[:person2_wca_id]).to include "This person has multiple subIds"
```

Comment:
This was testing `"Names don't match"` which is definitely not related to this test and comes from fact that the name of `person = FactoryGirl.create(:person_with_multiple_sub_ids)` was different from this person1's since we don't set it in the factory.
Note that in the spec above we didn't check this because inside MergePeople model the error about name difference is added to `:person2_wca_id`.